Windows 环境下的通配符错误

```shell
lint-md ./*
```
在 Windows 系统中使用该命令会报错
```shell
internal/fs/utils.js:220
    throw err;
    ^

Error: ENOENT: no such file or directory, stat 'C:\[DirectoryPath]\*.md'
```
而同样的命令在 Linux 系统或 Windows Subsystem for Linux 中都会正常展开通配符。
这是我的配置错误还是一个 BUG ？

以下是完整的 shell 信息和 log 文件，另外虽然不知道是否有关系，我在同样的位置使用 [Prettier](https://prettier.io/) 加通配符路径时并没有遇到问题
```shell
$ npm start

> @ start C:\Users\muzhou\Documents\GitHub\muzhou233.github.io
> lint-md ./*.md

internal/fs/utils.js:220
    throw err;
    ^

Error: ENOENT: no such file or directory, stat 'C:\Users\muzhou\Documents\GitHub\muzhou233.github.io\*.md'
    at Object.statSync (fs.js:915:3)
    at isDirectory (C:\Users\muzhou\Documents\GitHub\muzhou233.github.io\node_modules\lint-md-cli\bin\helper\file.js:22:13)
    at Timeout._onTimeout (C:\Users\muzhou\Documents\GitHub\muzhou233.github.io\node_modules\lint-md-cli\bin\helper\file.js:43:13)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7) {
  errno: -4058,
  syscall: 'stat',
  code: 'ENOENT',
  path: 'C:\\Users\\muzhou\\Documents\\GitHub\\muzhou233.github.io\\*.md'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ start: `lint-md ./*.md`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\muzhou\AppData\Roaming\npm-cache\_logs\2019-12-26T03_52_32_787Z-debug.log
```
[2019-12-26T03_52_32_787Z-debug.log](https://github.com/hustcc/lint-md/files/4001280/2019-12-26T03_52_32_787Z-debug.log)

--- a/packages/lint-md-cli/src/helper/file.js
+++ b/packages/lint-md-cli/src/helper/file.js
@@ -20,10 +20,13 @@ module.exports = (src, config) => {
   const { excludeFiles } = config;
 
   return new Promise((resolve, reject) => {
-    const srcArr = Array.isArray(src) ? src : [src];
-
     setTimeout(() => {
       const files = [];
+      const srcArr = [];
+        
+      for (let i = 0; i < src.length; i ++) {
+        srcArr.push(...glob.sync(src[i]))
+      }
 
       for (let i = 0; i < srcArr.length; i ++) {
         let f = [];

